opentelemetry-otlp/tests/integration_test/expected/traces.json

```diff
@@ -57,6 +57,12 @@
                   "value": {
                     "stringValue": "yes"
                   }
+                },
+                {
+                  "key": "data",
+                  "value": {
+                    "bytesValue": "gICA"
+                  }
```

Comment:
To make it complete, can we also add these in the json samples for logs and metrics?

Reply:
Sure yes!

Reply:
Also I wasn't sure how to add an example in a logs `body` field. Can you suggest the right way to do that?

Reply:
> Added. But if these are the outputs, shouldn't tests be failing?

Yes these would indeed fail. Please run ./scripts/integration_tests.sh locally to validate that. 

ps. - disable the failing tests from #2495 before that, so that you only get errors from your changes.

Reply:
> Also I wasn't sure how to add an example in a logs body field. Can you suggest the right way to do that?

Good question. The log appender doesn't [handle ](https://github.com/open-telemetry/opentelemetry-rust/blob/42b4f2fbd8c9be6abf749929f8b93299f077ddea/opentelemetry-appender-tracing/src/layer.rs#L121) bytes as of now. The good test probably would be adding binary data to [resource](https://github.com/open-telemetry/opentelemetry-rust/blob/42b4f2fbd8c9be6abf749929f8b93299f077ddea/opentelemetry-otlp/tests/integration_test/tests/logs.rs#L29).

Reply:
That's exactly what I did 😄 https://github.com/open-telemetry/opentelemetry-rust/pull/2471/files#diff-4629c7e6add9d0ba42d5f44077d079b4c6f85ac86bc326a42106d67146ae4dbc